corehq/apps/reports/filters/api.py

```diff
@@ -147,6 +148,16 @@ def get_groups(self, query, start, size):
         return [self.utils.reporting_group_tuple(g) for g in groups.run().hits]
 
 
+class LocationRestrictedEmwfOptionsView(EmwfOptionsView):
```

Comment:
Would follow your instinct here since i am limited by knowledge here and would leave this as-is for now. And i think if some report uses this filter it would need to be location safe, comes from the name itself `LocationRestricted`. :)